### ISSUE TITLE
Regular hover still shown when debugging

### ISSUE THREAD

**Issue by alexdima** [microsoft/vscode-jupyter#6435]
Testing https://github.com/microsoft/vscode/issues/127362

To be honest, I believe this is a VS Code core issue, but the TPI asks to create all issues here 🤷‍♂️ .

I think we normally hide the regular hover when doing debugging because of this overlap situation (cc @isidorn):

https://user-images.githubusercontent.com/5047891/123931154-a6aff700-d990-11eb-9fc0-45471f00b13d.mp4

**Comment by isidorn** [comment]
@alexdima correct, that is the motivation why we hide the regular hover (I think you even did that original code like 7 years ago :)

However note that we only hide the regular hover when we think there is a debug hover on the file (it is part of the stack frame). Since this is a special notebook debugging case probably some of the checks are off. I leave this to @rchiodo for an initial investigation and I can help if needed, just let me know.
